### ISSUE TITLE
Naming of zenodo sandbox cache for different branches

### ISSUE THREAD

**Issue by mathren** [showyourwork/showyourwork#108]
I have a `dev` branch of my repo where I am doing tests to progressively migrate to the updated `showyourwork`, and I created (following the [docs](https://show-your.work/en/latest/zenodo/#caching-the-intermediate-result) a cache on zenodo sandbox using `showyourwork cache create`. This however does not seem to create a very meaningful name:

![image](https://user-images.githubusercontent.com/49067756/172484614-838c9179-19a1-4536-bda1-dab0f3e74d79.png)

And in the autogenerated `zenodo.yml` file too I have:

```
cache:
  unknown:
    sandbox: 10.5072/zenodo.1070240
```
Feels like `[unknown]` here should be the branch name...

**Comment by mathren** [comment]
using the option `-b` and explicitly giving the branch name works though, so maybe another thing that can be fixed just editing the docs:
```
showyourwork cache create -b dev
```
produces the intended:

![image](https://user-images.githubusercontent.com/49067756/172488239-5dd1b130-94d1-4b9f-be14-bf682ae81da2.png)

**Comment by rodluger** [comment]
This could be related to #105. Can you tell me which version of `git` you're using?

**Comment by rodluger** [comment]
Specifically, if it's earlier than `2.22.0`, i know what's wrong, and the next dev release (tomorrow) will fix it!

**Comment by mathren** [comment]
It is...I'm on the cluster with an ancient git version

``` 
git --version
git version 1.8.3.1
```

**Comment by rodluger** [comment]
For now, see the solution in #105. Just use the latest commit hash as the version of showyourwork in your config file.

**Comment by rodluger** [comment]
Fixed in `v0.3.0.dev9`.
